vaadinfrontend/src/main/resources/application.properties

```diff
@@ -11,7 +11,7 @@ spring.datasource.url=${USER_DB_URL:localhost}
 spring.datasource.driver-class-name=${USER_DB_DRIVER:com.mysql.cj.jdbc.Driver}
 spring.datasource.username=${USER_DB_USER_NAME:myusername}
 spring.datasource.password=${USER_DB_USER_PW:astrongpassphrase!}
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
IMO this should not be the default setting since it can lead to clutter in the database 😨

Reply:
ah, leftover to update the data base with new tables for job runr and statistics.